core/Security/Password.php

```diff
@@ -142,7 +142,7 @@ private function _passExpired($uname) {
 	 */
 	public function getUserSalt($uname = '')
 	{
-		if (!isset($uname) || (isset($uname) && $uname == '')) {
```

Comment:
I think for this place, it would be better to use `empty($uname)` function. Less code similar result.

Reply:
you mean this ? : if (!isset($uname) || (isset($uname) && empty($uname) ))

Reply:
I mean you can replace all `!isset($uname) || (isset($uname) && $uname == '')` with one `empty($uname)` https://stackoverflow.com/a/15607428